src/App.tsx

```
@@ -1,26 +1,37 @@
+  const {
+    nodes,
+    edges,
+    onNodesChange,
+    onEdgesChange,
+    onConnect,
+    updateNode,
+    addNode,
+  } = useStore();
   const [selectedNode, setSelectedNode] = React.useState<string | null>(null);
-  const { project, getNodes, setViewport, zoomIn, zoomOut, fitView } = useReactFlow();
+  const { project, getNodes, setViewport, zoomIn, zoomOut, fitView } =
```

ESLint

[error] 'setViewport' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)